examples/BlackSky-product-example.json

```diff
@@ -0,0 +1,398 @@
+{
+    "products": [
+        {
+            "type": "Product",
+            "stat_version": "stat_version",
+            "stat_extensions": [],
+            "id": "BSKY_STEREO",
+            "title": "BlackSky EO STEREO",
+            "description": "A pair of images collected from different viewing angles for 3D visualization and analysis",
+            "keywords": [
+                "EO",
+                "Stereo"
+            ],
+            "license": "?",
+            "providers": {
+                "name": "BlackSky",
+                "description": "First To Know",
+                "roles": [
+                    "producer"
+                ],
+                "url": "www.blacksky.com"
+            },
+            "conformsTo": [
+                "https://geojson.org/schema/Point.json"
+            ],
+            "parameters": {
+                "$schema": "https://json-schema.org/draft-07/schema",
+                "type": "object",
+                "properties": {
+                    "eNiirs": {
+                        "type": "number",
+                        "description": "NIIRS Score is a function of the GSD, Signal-to-Noise Ratio (SNR), and Relative Edge Response (RER) of an image. This NIIRS constraint will use a predicted SNR and RER based off an aggregation of previously taken and analyzed images.",
+                        "minimum": 3,
+                        "maximum": 5
+                    },
+                    "target_azimuth": {
+                        "description": "Angle between True North, target, and sub-Sun point (clockwise)",
+                        "minimum": 0,
+                        "maximum": 360,
+                        "type": "integer"
```

Comment:
here and beow: I think it's "number" for json schemas instead of "integer" (https://json-schema.org/understanding-json-schema/reference/numeric#number)